aws/aws_instance.go

```diff
@@ -284,9 +285,47 @@ func (p *AWS) CreateInstance(ctx *lepton.Context) error {
 			addie := &ec2.InstanceIpv6Address{
 				Ipv6Address: aws.String(v6ad),
 			}
-			instanceInput.Ipv6Addresses = []*ec2.InstanceIpv6Address{addie}
+			instanceNIS.Ipv6Addresses = []*ec2.InstanceIpv6Address{addie}
 		} else {
-			instanceInput.SetIpv6AddressCount(1)
+			instanceNIS.SetIpv6AddressCount(1)
+		}
+	}
+
+	instanceInput.NetworkInterfaces = []*ec2.InstanceNetworkInterfaceSpecification{instanceNIS}
+
+	if cloudConfig.AwsIPN != "" || runConfig.AwsIPN != "" {
```

Comment:
is this actually required for scaling groups? I don't think so?

Reply:
Aws IntanceProfileName is optional